proposals/NNNN-Static-SDL-Icon-Names-Enum.md

```diff
@@ -0,0 +1,47 @@
+# Static SDL Icon Names Enum
+
+* Proposal: [SDL-NNNN](NNNN-Static-SDL-Icon-Names-Enum.md)
+* Author: [Nicole Yarroch](https://github.com/NicoleYarroch)
+* Status: **Awaiting review**
+* Impacted Platforms: iOS, Android
+
+## Introduction
+
+This proposal is to create a strongly typed enum for static SDL icon names. This will make it easier for developers to use these icons in their SDL apps.
+
+
+## Motivation
+
+Developers have access to a set of static on-board icons on SDL Core, however the developer must know the exact name of the icon in order to use it in a RPC. In addition, the static icons are obscurely named with hexadecimal values, which makes it hard to figure out which icon is currently being used. 
+
+## Proposed solution
+
+1. Create a new enum called `SDLStaticIconName`. The hexidecimal name for the static icon will be the string constant. 
+
+    ```objc
+    // SDLStaticIconName.h
+    typedef SDLEnum SDLStaticIconName SDL_SWIFT_ENUM;
+    extern SDLStaticIconName const SDLStaticImageNamePhoneDevice;
+    ```
+    ```objc
+    // SDLStaticIconName.h
+    SDLStaticIconName const SDLStaticIconName PhoneDevice = @“0x03”;
+    ```
+
+2. Add a convenience initializer to the `SDLImage` class that takes the new enum. 
+
+    ```objc
+    [SDLImage initWithStaticIconName:(SDLStaticIconName)]
+    ```
+
+## Potential downsides
+
+A manufacturer might choose to support a subset of the 100+ static icons. If a static icon is not supported, an empty icon will be used in its stead. Developers will still need to send a RPC using the static icon and then check the response from SDL Core to see if the static icon is actually available.
```

Comment:
I think the end of the first sentence should either be updated to "...used in its place." or "...used instead."

Reply:
@theresalech Fixed